src/components/presentations/NewArticle/BlockStyles/BlockStyleToolbar.jsx

```diff
@@ -0,0 +1,75 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import BlockStyleButton from './BlockStyleButton';
+import HeaderStyleDropdown from './HeaderStyleDropdown';
+
+export const BLOCK_TYPES = [
+  { label: ' “ ” ', style: 'blockquote' },
+  { label: 'UL', style: 'unordered-list-item' },
+  { label: 'OL', style: 'ordered-list-item' },
+];
+
+export const HEADER_TYPES = [
+  { label: 'H1', style: 'header-one' },
+  { label: 'H2', style: 'header-two' },
+  { label: 'H3', style: 'header-three' },
+  { label: 'H4', style: 'header-four' },
+  { label: 'H5', style: 'header-five' },
+  { label: 'H6', style: 'header-six' },
+];
+
+export function getBlockStyle(block) {
+  switch (block.getType()) {
+    case 'blockquote':
+      return 'RichEditor-blockquote';
+    default:
+      return null;
+  }
+}
```

Comment:
Can you define all these in a separate file and import them here? @AnayoOleru

Reply:
@rotimi-babalola I'll work on that now